latest/src/hooks/examples/useTextDataScanner.ts

```diff
@@ -25,18 +36,15 @@ export function useTextDataScanner() {
           unzoomedFinderHeight: 40,
         },
       };
-
-      setRtuTimeout(async () => {
-        await ScanbotSDK.UI.closeTextDataScanner();
-      });
       const result = await ScanbotSDK.UI.startTextDataScanner(config);
-      const data = result?.result?.text;
-      if (result.status !== 'OK' || !data) {
-        return;
+      /**
+       * Handle the result if result status is OK
+       */
+      if (result.status === 'OK' && result?.result?.text) {
```

Comment:
Please re-check the condition. Main result is mandatory and no need for null check